packages/frontend/src/hooks/useBigNumberConfig.ts

```diff
@@ -148,7 +158,42 @@ const useBigNumberConfig = (
               compact: bigNumberStyle,
           });
 
-    const showStyle = isNumber && (!isField(item) || item.format !== 'percent');
+    const formatValues = {
+        RAW: 'raw',
+        PERCENTAGE: 'percentage',
+    };
+    const [showComparison, setShowComparison] = useState<boolean>(false);
+    const [comparisonFormat, setComparisonFormat] = useState<string>(
+        formatValues.RAW,
+    );
+
+    const calculateComparisonValue = (a: number, b: number, format: string) => {
```

Comment:
We can move this function outside the hook since it doesn't depend on any external values, only on it's arguments.